cmd/cycles.go

```diff
@@ -17,11 +20,25 @@ var cyclesCmd = &cobra.Command{
 		chains := make(map[int][]string)
 		var temp []string
 		getChains(mainModule, depGraph, temp, chains, &cycleChains)
-		fmt.Println("All cycles in dependencies are: ")
 		cycles := getCycles(cycleChains)
 
-		for _, c := range cycles {
-			printChain(c)
+		if !jsonOutputCycles {
+			fmt.Println("All cycles in dependencies are: ")
+			for _, c := range cycles {
+				printChain(c)
+			}
+		} else {
+			// create json
+			outputObj := struct {
```

Comment:
ideally this PR should come with a test where I could see an example output as well to make the review easier :)

Reply:
Sorry I forgot that 😅. I'll add it in a separate PR :)